README.md

```diff
@@ -1,30 +1,23 @@
-# ion-blender-exporter
-
-Blender is a keystone of the 3D editing community, with the addition of the glTF exporter we wanted to make the process easier of getting your assets into [Cesium ion](https://cesium.com/). Anyone with a [Cesium ion](https://cesium.com/) account can take their models directly from Blender and have them available as 3DTiles within a matter of minutes. Whether the model is photogrammetry that was polished in Blender or is a 3D model of a city, ion can process that data to be available in an online and global context.
-
 <p align="center">
-    <a href="https://github.com/AnalyticalGraphicsInc/ion-blender-exporter/releases/latest">
-        <img width="250" src="https://i.imgur.com/V8Yxdb7.png">
-    </a>
+  <img src="https://github.com/AnalyticalGraphicsInc/cesium/wiki/logos/Cesium_Logo_Color.jpg" width="50%" />
 </p>
 
+The Cesium ion Blender add-on enables you to publish and stream even the most massive of 3D Models on the web with 3D Tiles.
+
+With 3D Tiles, even multi-gigabyte models can be streamed to any device without having to download the entire model up front. By loading 3D Tiles into CesiumJS, you can fuse your model with other datasets, add geospatial context to place it at a real world location, or overlay additional details and analysis.
```

Comment:
Is it better to just say "massive models" ? This kind of implies to me that "2-3 GB models" is kind of an upper limit on what 3D Tiles can do, which is untrue.

Reply:
I'm not sure Blender models grow that big, but I'll rephrase it.